app/_src/gateway/support/index.md

```diff
@@ -0,0 +1,87 @@
+
+
+## Sunset support
+After the product hits the end of the support period, Kong will provide limited support to help the customer upgrade to a fully supported version of {{site.ee_product_name}} for up to an additional 12 month sunset period. Kong will not provide patches for software covered by this sunset period. If there is an issue that requires a patch during this period, the customer will need to upgrade to a newer {{site.ee_product_name}} version covered by 24 month active support.
+
+## Bug fix guidelines
+Unfortunately, all software is susceptible to bugs. Kong seeks to remedy bugs through a structured protocol as follows:
+
+* Serious security vulnerabilities are treated with the utmost priority. See [here](/gateway/latest/production/security-update-process/) for our security vulnerability reporting and remedy process, including how to report a vulnerability.
+
+* Bugs which result in production outages of {{site.ee_product_name}} or effective non-operation (such as catastrophic performance degradation) will be remedied through high priority bug fixes and provided in patch releases to the Latest Major/Minor Version Release of all currently supported Major Versions of the software and optionally ported to other versions at Kong’s discretion based on the severity and impact of the bug.
+
+* Bugs which prevent the upgrade of a supported version of {{site.ee_product_name}} to a more recent supported version will be remedied through high priority bug fixes and provided in the Latest Major/Minor Version Release of all currently supported Major Versions of the software and optionally ported to other versions at Kong’s discretion based on the severity and impact of the bug.
+
+* Other bugs as well as feature requests will be assessed for severity and fixes or enhancements applied to versions of {{site.ee_product_name}} at Kong’s discretion depending on the impact of the bug. Typically, these types of fixes and enhancements will only be applied to the most recent Minor Version in the most recent Major Version.
+
+Customers with platinum or higher subscriptions may request fixes outside of the above and Kong will assess them at its sole discretion.
+
+## Supported versions
+
+Kong supports the following versions:
+
+{% navtabs %}
+  {% navtab 2.6 %}
+    {% include_cached gateway-support.html version="2.6" data=site.data.tables.support.gateway.versions.2x  eol="February 2023" %}
+  {% endnavtab %}
+    {% navtab 2.7 %}
+    {% include_cached gateway-support.html version="2.7" data=site.data.tables.support.gateway.versions.2x  eol="February 2023" %}
+  {% endnavtab %}
+  {% navtab 2.8 LTS %}
+    {% include_cached gateway-support.html version="2.8 LTS" data=site.data.tables.support.gateway.versions.2x  eol="March 2025" %}
+  {% endnavtab %}
+  {% navtab 3.0 %}
+    {% include_cached gateway-support.html version="3.0" data=site.data.tables.support.gateway.versions.3x eol="August 2023" %}
+  {% endnavtab %}
+  {% navtab 3.1 %}
```

Comment:
I think we should figure out how to make the default selected tab the current version of the docs.  Otherwise we should default to the latest version.  Currently when a visitor arrives at this page, they end up seeing the 2.6 docs which shows a lot more red X marks vs green checks that we support in the late model versions

Reply:
@rspurgeon, agreed and IIRC, proposed solution is to reorder versions so default is most current.

Reply:
resolved in latest